pages/about/index.md

```diff
@@ -2,36 +2,113 @@
+
+In doing so, we achieve:
+
+- Ensuring an open governance model where no single company dominates the specification and its tools.
+- Making the project as democratic as possible.
+- Relying on an asynchronous decision-making process that empowers the people who "work," not the companies who "pay." 
+
+In other words, it [gives equal power to individual and corporate contributors](/blog/governance-motivation).
 
 ## Maintainers 
 
-AsyncAPI is actively maintained by:
+**Committers** (also known as **Maintainers**) are the decision-makers at the repository/project level. They regularly contribute to the project and are invited by other maintainers to manage the repository, have more privileges, and approve pull requests.
 
-- [Fran Méndez](https://twitter.com/fmvilas) - Project Director.
-- [Łukasz Górnicki](https://twitter.com/derberq) - Maintainer and Dev Community Keeper.
+The committers run the repository, but for issues that go beyond that, there is a [Technical Steering Committee (TSC)](/community/tsc). This committee comprises all the maintainers who voluntarily want to be part of this committee, thus taking responsibility for overseeing the initiative.
 
-So far, we have had many contributors and we are thankful for their time and help.
+You can see the list of maintainers who also form part of the [AsyncAPI TSC here](/community/tsc).
 
 ## Contributors
 
 We do our best to recognize every contribution to the project. We do it individually in every repository from [AsyncAPI GitHub organization](https://github.com/asyncapi/). There should always be a **Contributors** section in the readme, like [this one](https://github.com/asyncapi/asyncapi/blob/master/README.md#contributors). We use [All Contributors](https://allcontributors.org/docs/en/specification) specification to handle recognitions. 
 
-We apologize in advance if we failed in recognizing your work. Feel free to contact us on [slack](https://www.asyncapi.com/slack-invite/) and we will fix it immediately, or just talk to [All Contributors](https://allcontributors.org) bot.
+We apologize in advance if we failed to recognize your work. Feel free to contact us on [Slack](https://www.asyncapi.com/slack-invite/), and we will fix it immediately, or just talk to our [All Contributors bot](https://allcontributors.org).
 
+## AsyncAPI, in numbers
+Interested to know more about our growth? Feel free to check our annual summary report: 
+- [Summary report 2020](/blog/2020-summary)
+- [Summary report 2021](/blog/2021-summary)
+          
 ## Companies support
 
 This is the list of companies that either provide a significant financial support or delegate their employees to support AsyncAPI with maintainance:
 
 <Sponsors className="mt-8" />
```

Comment:
please have a look at solace icon, it looks weirdly scaled

Reply:
@derberg, but I didnt change that Component file, should I try something to fix this or can I leave it as it is?

Reply:
@Harish-b-03 you are right, sorry, I didn't know it is broken on current website too. Feel free to just open separate issue for it 🙏🏼